core_lib_generator/template_core_lib/tests/test_template.py

```diff
@@ -2,11 +2,12 @@
 # template_core_lib_import
 # template_test_imports
 from core_lib.helpers.test import load_core_lib_config
+from core_lib_generator.template_core_lib.tests.test_data.helpers.util import sync_create_start_core_lib
 
 
 class TestTemplate(unittest.TestCase):
     def setUpClass(self):
-        config = load_core_lib_config('./test_data/test_config/config.yaml')
-        self.template_snake_core_lib = TemplateCoreLib(config)
+        self.template_snake_core_lib = sync_create_start_core_lib()

```

Comment:
can we also update the `core_lib_generator/file_generators/test_generator.py` to remove any updates we are not making